app/assets/stylesheets/mixins.scss

```
@@ -244,10 +244,12 @@
   width: calc(100% + #{$spacing});
   width: calc(100% + #{$max-spacing});
 
-  > * {
+  > *,
+  > .tabs-title {
     margin-bottom: 0;
     margin-left: $spacing;
     margin-left: $max-spacing;
+    margin-right: 0;
     margin-top: $spacing;
     margin-top: $max-spacing;
   }
```

There seems to be a padding added by Foundation to link tabs which is a bit annoying on `:focus` :relieved:.

```suggestion
  }
  
  a {
    padding: 0;
  }
```